### PR TITLE
fix: Degreed2 Missing Learner Data Audit Records

### PR DESCRIPTION
**Pull request by johnnagro**
## Description

- another missing model attribute
- clean up old debug logging

```
Mar 10 18:33:07 ip-10-2-70-10 [service_variant=lms][celery.app.trace][env:prod-edx-edxapp] ERROR [ip-10-2-70-10  1420] [user None] [ip None] [trace.py:265] - Task integrated_channels.integrated_channel.tasks.transmit_learner_data[b06ccff8-688e-4ada-b4f4-77d2e743f259] raised unexpected: AttributeError("'Degreed2LearnerDataTransmissionAudit' object has no attribute 'course_completed'")
Traceback (most recent call last):
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/celery/app/trace.py", line 451, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/newrelic/hooks/application_celery.py", line 99, in wrapper
    return wrapped(*args, **kwargs)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/celery/app/trace.py", line 734, in __protected_call__
    return self.run(*args, **kwargs)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/edx_django_utils/monitoring/internal/code_owner/utils.py", line 193, in new_function
    return wrapped_function(*args, **kwargs)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/integrated_channels/integrated_channel/tasks.py", line 109, in transmit_learner_data
    integrated_channel.transmit_learner_data(api_user)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/integrated_channels/integrated_channel/models.py", line 197, in transmit_learner_data
    transmitter.transmit(exporter)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/integrated_channels/degreed2/transmitters/learner_data.py", line 35, in transmit
    super().transmit(payload, **kwargs)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/integrated_channels/integrated_channel/transmitters/learner_data.py", line 267, in transmit
    if not learner_data.course_completed:
AttributeError: 'Degreed2LearnerDataTransmissionAudit' object has no attribute 'course_completed'
```

## References

- [ENT-5558](https://openedx.atlassian.net/browse/ENT-5558)

### CODE REVIEW

File: integrated_channels/integrated_channel/exporters/learner_data.py [hunk 1]
```diff
@@ -310,14 +310,6 @@ def get_grades_summary(
         else:
             completed_date_from_api, grade_from_api, is_passing_from_api, grade_percent = \
                 self.collect_grades_data(enterprise_enrollment, course_details, channel_name)
-            LOGGER.info(generate_formatted_log(
```
Comment (johnnagro) [inline integrated_channels/integrated_channel/exporters/learner_data.py:313]: added during a previous investigation, noisy and limited value after previous fix.

File: integrated_channels/integrated_channel/exporters/learner_data.py [hunk 2]
```diff
@@ -423,17 +415,6 @@ def export(self, **kwargs):
                 incomplete_count,
             )
 
-            LOGGER.info(generate_formatted_log(
```
Comment (johnnagro) [inline integrated_channels/integrated_channel/exporters/learner_data.py:426]: added during a previous investigation, noisy and limited value after previous fix.

File: integrated_channels/degreed2/models.py [hunk 3]
```diff
@@ -178,6 +178,11 @@ class Degreed2LearnerDataTransmissionAudit(models.Model):
         help_text="Used as content-id field when creating a completion"
     )
 
+    course_completed = models.BooleanField(
```
Comment (johnnagro) [inline integrated_channels/degreed2/models.py:181]: Cribbed from the Degreed1 version. Required by this code:

https://github.com/edx/edx-enterprise/blob/jnagro/ENT-5558/1/integrated_channels/integrated_channel/transmitters/learner_data.py#L267

Another example where a standard parent class for LearnerDataTransmissionAudit models would help us.

**Review by binodpant — APPROVED** [review]
ty for fixing this!

**Pull request merged**